nullaway/src/test/java/com/uber/nullaway/NullAwayContractsTests.java

```diff
@@ -42,9 +42,9 @@ public void basicContractAnnotation() {
             "import javax.annotation.Nullable;",
             "import org.jetbrains.annotations.Contract;",
             "public class NullnessChecker {",
-            "  @Contract(\"_, null -> true\")",
+            "  @Contract(\"_, null -> true; _, !null -> false\")",
```

Comment:
Both this contract and the example on line 47 are an example of the overzealous assumption I described in the PR description. A `null -> false` doesn't provide much insight into `!null -> [value]`, both can result in false. I added a more interesting test case below where a method in which either of two args is null returns false, which failed prior to this change.

I would have preferred to make this change in a separate PR, but it would have certainly conflicted with the work toward #667 (and is required for other new tests to work correctly). Regardless, I can split this piece into a separate PR if you prefer, or if the boolean contract piece requires a longer discussion. :-)

Reply:
I was incorrect here